src/ViewLayer/DebugDisplay/MotorFaultPage/MotorFaultView/MotorFaultView.cpp

```diff
@@ -0,0 +1,372 @@
+    , chargerSafetyRelayFault_ ("Charger Safety Relay Fault")
+    , currentSensorFault_ ("Current Sensor Fault")
+    , dischargeLimitEnforcementFault_ ("Discharge Limit Enforcement Fault")
+    , fanMonitorFault_ ("Fan Monitor Fault")
+    , highVoltageIsolationFault_ ("High Voltage Isolation Fault")
+    , internalCommununicationFault_ ("Internal Communication Fault")
+    , internalConversionFault_ ("Internal Conversion Fault")
+    , internalLogicFault_ ("Internal Logic Fault")
+    , internalMemoryFault_ ("Internal Memory Fault")
+    , internalThermistorFault_ ("Internal Thermistor Fault")
+    , lowCellVoltageFault_ ("Low Cell Voltage Fault")
+    , openWiringFault_ ("Open Wiring Fault")
+    , packVoltageSensorFault_ ("Pack Voltage Sensor Fault")
+    , powerSupplyFault12V_ ("12V Power Supply Fault")
+    , thermistorFault_ ("Thermistor Fault")
+    , voltageRedundancyFault_ ("Voltage Redundancy Fault")
+    , weakCellFault_ ("Weak Cell Fault")
+    , weakPackFault_ ("Weak Pack Fault")
+    , cclReducedDueToAlternateCurrentLimit_ ("CCL Reduced Due To Alternate Current")
+    , cclReducedDueToChargerLatch_ ("CCL Reduced Due To Charger Latch")
+    , cclReducedDueToHighCellResistance_ ("CCL Reduced Due To High Cell Resistance")
+    , cclReducedDueToHighCellVoltage_ ("CCL Reduced Due To High Cell Voltage")
+    , cclReducedDueToHighPackVoltage_ ("CCL Reduced Due To High Pack Voltage")
+    , cclReducedDueToHighSoc_ ("CCL Reduced Due To High SOC")
+    , cclReducedDueToTemperature_ ("CCL Reduced Due To Temperature")
+    , dclandCclReducedDueToCommunicationFailsafe_ ("DCL and CCL Reduced Due To Communication Fail Safe")
+    , dclandCclReducedDueToVoltageFailsafe_ ("DCL and CCL Reduced Due To Voltage Fail Safe")
+    , dclReducedDueToHighCellResistance_ ("DCL Reduced Due To High Cell Resistance")
+    , dclReducedDueToLowCellVoltage_ ("DCL Reduced Due To Low Cell Voltage")
+    , dclReducedDueToLowPackVoltage_ ("DCL Reduced Due To Low Pack Voltage")
+    , dclReducedDueToLowSoc_ ("DCL Reduced Due To Low SOC")
+    , dclReducedDueToTemperature_ ("DCL Reduced Due To Temperature")
+    , labelBCount_ (0)
+{
+
+    // Setting up Vertical bar
+    QScrollBar* verticalBar0 = new QScrollBar();
+    QScrollBar* verticalBar1 = new QScrollBar();
+    QScrollBar* verticalBarB = new QScrollBar();
+
+    verticalBar0->setStyleSheet(SCROLLBAR_STYLESHEET);
+    verticalBar1->setStyleSheet(SCROLLBAR_STYLESHEET);
+    verticalBarB->setStyleSheet(SCROLLBAR_STYLESHEET);
+
+    ui_.motor0ScrollArea().setVerticalScrollBar(verticalBar0);
+    ui_.motor1ScrollArea().setVerticalScrollBar(verticalBar1);
+    ui_.batteryScrollArea().setVerticalScrollBar(verticalBarB);
+
+    QLayout* layout0 = ui_.motor0ContentsWidget().layout();
+    QLayout* layout1 = ui_.motor1ContentsWidget().layout();
+    QLayout* layoutB = ui_.batteryContentsWidget().layout();
+
+    // Motor 0
+    initializeLabel(badMotorPositionHallSequence0Fault_, layout0, ERROR_STYLESHEET);
```

Comment:
Now that I think more about it, it looks kinda cancer to have 100 lines of the same function in the constructor. Encapsulate all of this in an `initializeLabels()` function.